lib/Elevate/Blockers/Databases.pm

```diff
@@ -40,6 +46,71 @@ sub _warning_if_postgresql_installed ($self) {
     return 2;
 }
 
+sub _blocker_acknowledge_postgresql_datadir ($self) {
+
+    return 0 unless Cpanel::Pkgr::is_installed('postgresql-server');
+
+    my $touch_file = POSTGRESQL_ACK_TOUCH_FILE;
+    return 0 if -e $touch_file;
+
+    my @users_with_dbs = $self->_has_mapped_postgresql_dbs();
+    return 0 unless scalar @users_with_dbs;
+
+    my $message = <<~"EOS";
+    One or more users on your system have associated PostgreSQL databases.
+    ELevate may upgrade the software packages associated with PostgreSQL
+    automatically, but if it does, it will *NOT* automatically update the
+    PostgreSQL data directory to work with the new version. Without an update
```

Comment:
Can't we run the script to upgrade them? Why can't we use the psql channel to switch them to 8.3 instead of 10?

Reply:
It can't be 8.3 because downgrades are not possible.

Reply:
Sorry I meant 9.6 See these old instructions:

```
-    You have postgresql-server version 9.2 installed.
-    This is upgraded irreversably to version 10.0 when you switch to AlmaLinux 8
-    We recommend data backup and removal of all postgresql packages before upgrade to AlmaLinux 8.
-    To re-install postgresql 9 on AlmaLinux 8, you can run: `dnf -y module enable postgresql:9.6; dnf -y install postgresql-server`
```

Reply:
In theory we could do this for them...

Reply:
Yes, but we would still need to perform the data directory update, because prior to PostgreSQL 10, the first two elements of the version number are considered the "major" version: 9.2.x to 9.6.x is a major upgrade.